models/response_model.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+type PageDescription struct {
+	Size          int `json:"size"`
+	Number        int `json:"number"`
+	TotalElements int `json:"totalElements"`
+	TotalPages    int `json:"totalPages"`
+}
+
+type PageModel[T any] struct {
+	Content []*T            `json:"content"`
```

Comment:
Maybe `data` would be a better name. WDYT?

Reply:
I'm not sure - both `content` and `data` seem correct. `content` was used in the previous PRs